google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITNotificationTest.java

```diff
@@ -190,8 +187,7 @@ public void testNotification() throws InterruptedException, ExecutionException {
           .isNull();
       assertThat(storage.listNotifications(bucketFixture.getBucketInfo().getName())).isEmpty();
     } finally {
-      RemoteStorageHelper.forceDelete(
-          storage, bucketFixture.getBucketInfo().getName(), 5, TimeUnit.SECONDS);
+      // delete is taken care of by BucketFixture now
```

Comment:
Are you oging to remove the finally block?

Reply:
Probably not, because then the code formatter will change all the indentation for the whole test and git-blame me on those lines 😄